src/components/containers/main/index.js

```diff
@@ -0,0 +1 @@
+export { default } from './main'
```

Comment:
Why dont you use just main, what is the reason of creation new files?

Reply:
okay I remove this component and use <main>